tuna/fin_class.py

```diff
@@ -177,7 +176,7 @@ def parse_out(self):
         self.logger.warning('Err loading fin json: %s', err)
         return None
     else:
-      with open(self.local_output) as out_file:
+      with open(self.local_output) as out_file: # pylint: disable=unspecified-encoding
```

Comment:
what's the encoding for the json file? I can add it, instead of silencing the warning. (utf-8?)